gke-deploy/cmd/apply/apply.go

```diff
@@ -58,14 +58,14 @@ func NewApplyCommand() *cobra.Command {
 		SilenceUsage: true,
 	}
 
-	cmd.Flags().StringVarP(&options.filename, "filename", "f", "", "Configuration file or directory of configuration files to use to create Kubernetes objects (file or files in directory must end with \".yml\" or \".yaml\").")
+	cmd.Flags().StringVarP(&options.filename, "filename", "f", "", "Configuration file or directory of configuration files to use to create Kubernetes objects (file or files in directory must end with \".yml\" or \".yaml\"). The configuration file(s) will be download to a temporary folder first if the file or directory is a GCS URL, e.g. gs://bucket/directory/*.")
```

Comment:
Maybe:

Local or GCS path to configuration file or directory of configuration files to use to create Kubernetes objects (file or files in directory must end in \".yml\" or \".yaml\"). Prefix this value with \"gs://\" to indicate a GCS path.

Reply:
@joonlim , your suggestion here leaves out the information that file/s from GCS are downloaded to a temp dir first. Is it ok to leave that out?

Reply:
Yes, I think the user doesn't have to know that that's happening behind the scenes

Reply:
Copy-pasted :)